package.json

```diff
@@ -92,6 +92,7 @@
     "istanbul": "^1.1.0-alpha.1",
     "jsdom": "^9.11.0",
     "mocha": "^3.2.0",
+    "npm-run-all": "^4.0.2",
```

Comment:
Used for [`run-s`](https://github.com/mysticatea/npm-run-all/blob/master/docs/run-s.md) in the CircleCI config in https://github.com/pyrolabs/overwire/pull/7/files/810056e4c6372be6997a52adffa477d13e4865e6#diff-29944324a3cbf9f4bd0162dfe3975d88R20